### PR TITLE
Lsf kdd final

### PR DESCRIPTION
**Pull request by harishB97**
Modified the data file path.

In heatmapLSF.py, reverted the code back to the one that was used for generating the heatmap shown in the paper. Also updated the readme.md accordingly.

heatmapLSF.yaml in analysis/configs can be deleted for now, it is not used by any code.

### CODE REVIEW

File: analysis/heatmapLSF.py [hunk 1]
```diff
@@ -1,89 +1,155 @@
...
-        sort_indices = np.argsort(classes)
-        sorted_encodings = encodings[sort_indices,:]
-        sorted_classes = sorted(classes)
+    def __getitem__(self, idx):
+        return self.data[idx]
 
-        z_cosine_distances = get_CosineDistance_matrix(sorted_encodings)
 
-        os.makedirs(plot_save_dir, exist_ok=True)
+class DataModuleFromConfig(pl.LightningDataModule):
```
Comment (elhamod) [inline analysis/heatmapLSF.py]: This function already exists in main. cant we just import it?

Comment (elhamod) [reply analysis/heatmapLSF.py]: same with other classes in this file

File: analysis/configs/tsneLSF.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ model_name: 'LSF'
 
 file_prefix: "LSF-test-set"
 img_res: 256
-dataset_path: "/fastscratch/elhamod/data/Fish/taming_transforms_fish_test_padded_256_imagenetmeancolor.txt"
+dataset_path: "/fastscratch/elhamod/data/Fish/fish_test.txt"
 
 ckpt_path: /home/harishbabu/projects/taming-transformers/logs/2023-02-01T23-38-48_LSF2-VQVAE1024emb-256img_imagenetmean_noaug-base_noaug_withimgnet/checkpoints/last.ckpt
 yaml_path: /home/harishbabu/projects/taming-transformers/logs/2023-02-01T23-38-48_LSF2-VQVAE1024emb-256img_imagenetmean_noaug-base_noaug_withimgnet/configs/2023-02-01T23-38-48-project.yaml
```
Comment (elhamod) [inline analysis/configs/tsneLSF.yaml]: rename the models to be userfriendly and upload the same name to google drive. take a look at my yamls for examples

**Review by elhamod — CHANGES_REQUESTED** [review]

File: analysis/heatmapLSF.py (hunk 1 above)
Comment (harishB97) [reply analysis/heatmapLSF.py]: Modified Moe

File: analysis/configs/tsneLSF.yaml (hunk 2 above)
Comment (harishB97) [reply analysis/configs/tsneLSF.yaml]: Updated the model paths. Should we also change the absolute paths to relative paths like
"/home/harishbabu/projects/phylonn/logs/LSF/checkpoints/last.ckpt" to "./logs/LSF/checkpoints/last.ckpt"

**Review by elhamod — APPROVED** [review]

**Pull request merged**